docs/resources/fgs_function.md

```diff
@@ -328,6 +328,11 @@ The `func_mounts` block supports:
 
 * `local_mount_path` - (Required, String) Specifies the function access path.
 
+* `concurrency_num` - (Optional, String) Specifies the number of concurrent requests of the function.
+  The valid value ranges from `-1` to `1,000`.
```

Comment:
The description of the valid values are missing?
According to the maximum instance number

Reply:
API doc range from **-1** to  **1000**
If the parameter is not specified , the default value is  **1**